### PR TITLE
Initial documentation for Mailer bundle and component

### PR DESCRIPTION
**Pull request by pjedrzejewski**

### CODE REVIEW

File: bundles/SyliusMailerBundle/index.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+SyliusMailerBundle
+==================
+
+Sending customizable e-mails has never been easier in Symfony.
+
+You can configure different e-mail types in the YAML or in database. (and use YAML as fallback)
+This allows you to send out e-mails with one simple method call, providing an unique code and data.
+
+The bundle supports adapters, by default e-mails are rendered using Twig and sent via Swiftmailer, but you can easily implement your own adapter nad delegate the whole operation to external API.
```
Comment (amenophis) [inline bundles/SyliusMailerBundle/index.rst]: nad => and

File: bundles/SyliusMailerBundle/your_first_email.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
+Your First Email
+================
+
+Let's say you want to send a notification to the website team when someone submits a new position to your move catalog website!
```
Comment (amenophis) [inline bundles/SyliusMailerBundle/your_first_email.rst]: move => movie

File: bundles/SyliusMailerBundle/configuration.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+=======================
+
+.. code-block:: yaml
+
+    sylius_mailer:
+          driver: ~ # The driver used for persistence layer. Currently only `doctrine/orm` is supported.
+          adapter: twig_swiftmailer # Adapter for sending e-mails.
+          classes:
+              email:
+                  model:      Sylius\Component\Mailer\Model\Email # The archetype model class implementing `ArchetypeInterface`.
```
Comment (amenophis) [inline bundles/SyliusMailerBundle/configuration.rst]: Archetype ? really ?

**Comment by pjedrzejewski** [comment]
Fixed, thanks @amenophis! :+1:

**Pull request merged**